ROLES.md

```diff
@@ -232,31 +232,44 @@ approver in an OWNERS file:
 
 ## Lead
 
-Working group leads, or just ‘leads’, are approvers of an entire area that have
-demonstrated good judgement and responsibility. Leads accept design proposals
-and approve design decisions for their area of ownership.
+Working group leads, or just ‘leads’, coordinate working group processes and
+decision-making. Leads are responsible for selecting appropriate processes and
+tracking and reporting on the working group's results to the TOC. Leads
+collaboratively take responsibilty for accepting design proposals and approving
```

Comment:
This isn't clear on whether they're a mini-dictator or more like a moderator helping to group come to a consensus. I'd prefer the latter.

Reply:
This is intended to suggest the latter (the word "collaboratively" is intended to suggest this). They are also responsible for delivering results of the process, so at some point there may come a "disagree and commit" decision that the leads (collectively) are responsible for.

Would adding a sentence like "A healthy working group has multiple leads, though not so many that decision making becomes difficult." help?